src/components/SettingsButton/index.tsx

```diff
@@ -8,7 +8,10 @@ import { Button } from '@aiera/client-sdk/components/Button';
 import { useAuthContext } from '@aiera/client-sdk/lib/auth';
 import './styles.css';
 
-interface SettingsButtonSharedProps {}
+interface SettingsButtonSharedProps {
+    showTextSentiment?: boolean;
```

Comment:
These can also just be `showSentiment` or `hideSentiment` if we don't care to split those up.

Reply:
I think keeping them separated is probably good.